pkg/benchmarktiming/benchmarktiming_suite_test.go

```diff
@@ -41,7 +41,7 @@ func TestDispatch(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	numWorkers = 4
-	testFunc = "../../resources/functions/jsondump.py"
+	testFunc = "/Users/nkaashoek/go/src/github.com/benchmark/resources/functions/jsondump.py"
```

Comment:
This looks like some leftover (others won't have the same path). See other files for similar issues